templates/rust-starter/README.md

```diff
@@ -23,6 +23,16 @@ cargo run
 This is an empty template, and so there is no expected output (until you modify
 the code).
 
+### Executing the project locally in development mode
+
+During development, faster iteration upon code changes can be achieved by leveraging [dev-mode], we strongly suggest activating it during your early development phase. Furthermore, you might want to get insights into the execution statistics of your project, and this can be achieved by specifying the environment variable `RUST_LOG=RISC0_EXECUTION_STATS` before running your project.
+
+Put together, the command to run your project in development mode while getting execution statistics is:
+
+```bash
+RUST_LOG=RISC0_EXECUTION_STATS RISC0_DEV_MODE=1 cargo run
```

Comment:
Can you still set your logging level when using `RISC0_EXECUTION_STATS` ?

Reply:
Yes, it's possible. And I changed to conform to your suggestion of `RUST_LOG="executor=info"` :)